aikau/src/main/resources/alfresco/renderers/EditableComment.js

```diff
@@ -236,7 +236,7 @@ define(["dojo/_base/declare",
        * @instance
        * @param {object} payload The details of the comment update
        */
-      onEditSave: function alfresco_renderers_EditableComment__onEditSave(payload) {
+   onEditSave: function alfresco_renderers_EditableComment__onEditSave(payload) {
```

Comment:
This looks like an outdent that then mis-aligns the function definition ... is that right?